kernels/quantized/cpu/op_embedding2b.cpp

```diff
@@ -0,0 +1,350 @@
+/*
```

Comment:
rename op_embedding4b.cpp to op_embedding_nbit.cpp and have 2b/4b in the same file. I suspect there is a lot of code reuse opportunity